avm/Cargo.toml

```diff
@@ -12,7 +12,10 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-ascii_table = "3.0"
+# New commits to a git dependency are locked in place and only replaced by a `cargo update`.
+anycloud = { git = "https://github.com/alantech/anycloud", branch = "main" }
+# For local development, it is possible to specify a relative path location instead of a git repository
+# anycloud = { path = "../../anycloud/cli" }
```

Comment:
Why does this one have `/cli` at the end but not the git repo above?

Reply:
The one above is the github repository and finds the cargo lib recursively, no need (or way) to specify the folder. That one below however specifies a specific folder path.

Reply:
That's bonkers. What if there are two cargo libs in the repo?

Reply:
I agree and had the same thought